Commons/src/main/java/hk/hku/cecid/piazza/commons/util/StringUtilities.java

```diff
@@ -30,7 +25,8 @@
 
     private static final String DEFAULT_VALUE_SEPARTOR = ":";
     public final static String LINE_SEPARATOR = System.getProperty("line.separator", "\n");
-    
+    protected static org.apache.log4j.Logger logger = org.apache.log4j.Logger.getLogger(StringUtilities.class);
```

Comment:
this logger isn't used is it needed...

Also please add unit test to test the environment variable handling